Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -425,7 +425,9 @@ public class Fuzzer {
                     for i in stride(from: lineNumber, to: lineNumber + 2 * instrCount, by: 2) {
                         let proto = try Fuzzilli_Protobuf_Type(jsonUTF8Data: lines[i+1].data(using: .utf8)!)
                         let runtimeType = try Type(from: proto)
-                        program.types.setType(of: variable, to: runtimeType, after: Int(lines[i])!, quality: .runtime)
+                        if runtimeType != .unknown {
```

Comment:
Maybe add a small comment here explaining in which case we'd get `.unknown` here?